src/component/ChairMapping/ChairMapping.js

```diff
@@ -0,0 +1,63 @@
+import {MdArrowBackIos} from 'react-icons/md';
+import {useParams, useNavigate} from 'react-router-dom';
+import styled from 'styled-components';
+
+import ChairCard from '../ChairCard/ChairCard';
+
+export default function ChairMapping({chairsData, toggleDescription, showDescription}) {
+  const {name} = useParams();
+  let navigate = useNavigate();
+
+  function handleClick() {
+    navigate('/');
+  }
```

Comment:
I would suggest not to write a function for navigating back to the main page. You can simply do this directly in the `onClick` of your button:

`  <button onClick={() => navigate('/')}>`